x/ccv/consumer/ibc_module_test.go

```diff
@@ -262,7 +261,7 @@ func TestOnChanOpenAck(t *testing.T) {
 		{
 			"invalid: mismatched serialized version",
 			func(keeper *consumerkeeper.Keeper, params *params, mocks testkeeper.MockedKeepers) {
-				md := providertypes.HandshakeMetadata{
+				md := ccv.HandshakeMetadata{
```

Comment:
Why do we have `ccv "github.com/cosmos/interchain-security/v3/x/ccv/types"` here and  not simply`"github.com/cosmos/interchain-security/v3/x/ccv/types"` as in `ibc_module.test.go`. Does this lead to inconsistencies between `ibc_module.go` and `ibc_module_test.go`, e.g., we have `ccv.HandshakeMetadata` here instead of `types.HandshakeMetadata` in `ibc_module.go`?

Reply:
See https://github.com/cosmos/interchain-security/pull/1071#discussion_r1296133443, the type aliases in this repo are very inconsistent and this does lead to inconsistencies, but I don't think it's worth fixing all in this PR

Reply:
👍 I agree. We can create another PR for fixing naming inconsistencies across _all_ the files, just so we don't forget.